src/main/java/it/pagopa/ecommerce/commons/documents/v2/TransactionRefundRequestedData.java

```diff
@@ -0,0 +1,46 @@
+package it.pagopa.ecommerce.commons.documents.v2;
+
+import it.pagopa.ecommerce.commons.documents.v2.authorization.TransactionGatewayAuthorizationData;
+import it.pagopa.ecommerce.commons.generated.server.model.TransactionStatusDto;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Generated;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import javax.annotation.Nullable;
+
+/**
+ * Data class that contains information about a transaction for which a refund
+ * operation have been requested
+ *
+ * @see BaseTransactionRefundedData
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+@Document
+@NoArgsConstructor
+@Generated
+public final class TransactionRefundRequestedData extends BaseTransactionRefundedData {
+
+    /**
+     * Gateway authorization data, as retrieved during refund operation
```

Comment:
I would add when it can be null (e.g. if there is no auth completed we retrieve the data)